ci/terraform/dns.tf

```diff
@@ -8,10 +8,11 @@ locals {
 
   account_management_fqdn = var.environment == "production" ? "home.account.gov.uk" : "home.${var.environment}.account.gov.uk"
 
-  frontend_fqdn        = "signin.${local.service_domain}"
-  frontend_api_fqdn    = "auth.${local.service_domain}"
-  oidc_api_fqdn        = "oidc.${local.service_domain}"
-  frontend_fqdn_origin = "origin.signin.${local.service_domain}"
+  frontend_fqdn                = "signin.${local.service_domain}"
+  frontend_api_fqdn            = "auth.${local.service_domain}"
+  oidc_api_fqdn                = "oidc.${local.service_domain}"
+  frontend_fqdn_origin         = "origin.signin.${local.service_domain}"
+  frontend_service_down_origin = "origin2.signin.${local.service_domain}"
 }
```

Comment:
I have knowledge gaps in how DNS works in these accounts, but spotted `origin2`. Is this correct or a typo here?

Reply:
Removed as this was Placeholder for cloudfront